crates/torii/migrations/20240829114436_merge_worlds_and_indexer.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE contracts (
+    -- contract_address
+    id TEXT NOT NULL PRIMARY KEY,
+    contract_address TEXT NOT NULL,
```

Comment:
Duplicate, we could only keep the id as being the address.

Reply:
same reason as: https://github.com/dojoengine/dojo/pull/2373#discussion_r1739580540